x-pack/plugins/lists/common/schemas/request/update_exception_list_item_schema.test.ts

```diff
@@ -0,0 +1,183 @@
+    const outputPayload = getUpdateExceptionListItemSchemaMock();
+    delete inputPayload.comments;
+    outputPayload.comments = [];
+    const decoded = updateExceptionListItemSchema.decode(inputPayload);
+    const checked = exactCheck(inputPayload, decoded);
+    const message = pipe(checked, foldLeftRight);
+    expect(getPaths(left(message.errors))).toEqual([]);
+    expect(message.schema).toEqual(outputPayload);
+  });
+
+  test('it should accept an undefined for "entries" but return an array', () => {
+    const inputPayload = getUpdateExceptionListItemSchemaMock();
+    const outputPayload = getUpdateExceptionListItemSchemaMock();
+    delete inputPayload.entries;
+    outputPayload.entries = [];
+    const decoded = updateExceptionListItemSchema.decode(inputPayload);
+    const checked = exactCheck(inputPayload, decoded);
+    const message = pipe(checked, foldLeftRight);
+    expect(getPaths(left(message.errors))).toEqual([]);
+    expect(message.schema).toEqual(outputPayload);
+  });
+
+  test('it should accept an undefined for "namespace_type" but return enum "single"', () => {
+    const inputPayload = getUpdateExceptionListItemSchemaMock();
+    const outputPayload = getUpdateExceptionListItemSchemaMock();
+    delete inputPayload.namespace_type;
+    outputPayload.namespace_type = 'single';
+    const decoded = updateExceptionListItemSchema.decode(inputPayload);
+    const checked = exactCheck(inputPayload, decoded);
+    const message = pipe(checked, foldLeftRight);
+    expect(getPaths(left(message.errors))).toEqual([]);
+    expect(message.schema).toEqual(outputPayload);
+  });
+
+  test('it should accept an undefined for "tags" but return an array', () => {
+    const inputPayload = getUpdateExceptionListItemSchemaMock();
+    const outputPayload = getUpdateExceptionListItemSchemaMock();
+    delete inputPayload.tags;
+    outputPayload.tags = [];
+    const decoded = updateExceptionListItemSchema.decode(inputPayload);
+    const checked = exactCheck(inputPayload, decoded);
+    const message = pipe(checked, foldLeftRight);
+    expect(getPaths(left(message.errors))).toEqual([]);
+    expect(message.schema).toEqual(outputPayload);
+  });
+
+  test('it should accept an undefined for "_tags" but return an array', () => {
+    const inputPayload = getUpdateExceptionListItemSchemaMock();
+    const outputPayload = getUpdateExceptionListItemSchemaMock();
+    delete inputPayload._tags;
+    outputPayload._tags = [];
+    const decoded = updateExceptionListItemSchema.decode(inputPayload);
+    const checked = exactCheck(inputPayload, decoded);
+    const message = pipe(checked, foldLeftRight);
+    expect(getPaths(left(message.errors))).toEqual([]);
+    expect(message.schema).toEqual(outputPayload);
+  });
+
+  // TODO: Is it expected behavior for it not to auto-generate a uui or throw
+  // error if item_id is not passed in?
+  xtest('it should accept an undefined for "item_id" and auto generate a uuid', () => {
```

Comment:
@FrankHassanabad I think I might have seen a to do related to this, but just wanted to double check the expected behavior here.